src/io/flutter/FlutterBundle.properties

```diff
@@ -63,3 +63,10 @@ flutter.run.bazel.noLaunchingScript=no launching script specified
 flutter.run.bazel.launchingScriptNotFound=launching script not found: {0}
 flutter.run.bazel.noTargetSet=no Bazel target set
 flutter.run.bazel.noWorkingDirectorySet=no working directory set
+
+flutter.analytics.notification.title=Welcome to Flutter!
+flutter.analytics.notification.content=The Flutter plugin anonymously reports feature usage statistics and basic crash reports to Google \
```

Comment:
Leave out the word 'basic' in 'basic crash reports' (per the phrase approved in the PDD)